src/Stratis.Bitcoin/Consensus/ConsensusManger.cs

```diff
@@ -379,8 +382,9 @@ private async Task<(ValidationContext ValidationContext, bool ConsensusTipChaneg
                 // In that case we try to load the blocks from store, if store is not present
                 // we mark the blocks for download and do not proceed.
 
-                if (oldTip.Height - current.Height > 1)
+                if (current != fork)
```

Comment:
yes correct this looks better.
But now the line bellow ` fork = current;` can go inside the `if`